Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/AutoFramework.java

```diff
@@ -0,0 +1,99 @@
+package org.firstinspires.ftc.team6220_PowerPlay;
+
+
+abstract public class AutoFramework extends BaseAutonomous {
+
+    /**
+     * @param AutoSelector Assign to 0 or 1, corresponding to BlueLeft/RedLeft, or BlueRight/RedRight respectively
+     * @throws InterruptedException If you haven't added a config for that autonomous
+     */
+    public void runAuto(int AutoSelector) throws InterruptedException {
+
+
+        //set values here
+        int driveCourse;
+        int targetDistance = 11;
+        int[] signalArray;
+        switch (AutoSelector) {
+
+            //BlueLeft & RedLeft
```

Comment:
Does anything color specific happen or is it really just left and right?

Reply:
it's just left and right, made the change